macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -318,6 +346,24 @@ open class AvatarView: NSView {
 		initialsFontColor = color.foreground.resolvedColor(appearance)
 	}
 
+	private func updateHeight() {
+		let diameter = diameterForContentCircle()
+		let circularPath = CGPath.circularPath(withCircleDiameter: diameter)
+		circleMask.path = circularPath
+
+		let textFieldFont = font(forCircleDiameter: diameter)
+		initialsTextField.font = textFieldFont
+
+		// This constraint will only exist if we're using the sizing view approach to center our initials
+		initialsSizingViewHeightConstraint?.constant = ceil(textFieldFont.capHeight)
+		contentViewHeightConstraint?.constant = diameter
+		contentViewWidthConstraint?.constant = diameter
+	}
+
+	private func diameterForContentCircle() -> CGFloat {
+		return hasBorder ? avatarSize - (AvatarView.borderWidth * 2) - (AvatarView.contentInset * 2) : avatarSize
```

Comment:
Not sure if the design spec is clear on this behavior, but for the iOS Vnext the avatar size is always the same and the ring and gap/inset add to its size instead of subtracting. (if the goal is consistency across platforms)

Reply:
Not sure if relevant but when I asked Kaan if the size ramp on avatar for iOS and macOS should be the same, he said yeah. So we probably want to match?

Reply:
that's interesting. so no longer avatar vnext will be 72 x 72 it will be 80 x 80 if there is a border ring? but again i am following existing iOS avatar calculation than vnext for sure.

Reply:
following up with design team

Reply:
i think the real direction should come once vnext Swiftui avatar comes to macOS this fall and we will actually have the sizeRamp for macOS avatar view which we don't right now. So, to keep avatarSize intact, i will do the subtraction rather than addition direction.